### DISCUSSION TITLE
CVE-2021-26086.yaml

### DISCUSSION THREAD

**Discussion by shelu16** [projectdiscovery/nuclei-templates#3674] [General]
CVE-2021-26086.yaml

I think you need to add the below line also

WEB-INF/web.xml
WEB-INF/decorators.xml
WEB-INF/classes/seraph-config.xml
META-INF/maven/com.atlassian.jira/jira-webapp-dist/pom.properties
META-INF/maven/com.atlassian.jira/jira-webapp-dist/pom.xml
META-INF/maven/com.atlassian.jira/atlassian-jira-webapp/pom.xml
META-INF/maven/com.atlassian.jira/atlassian-jira-webapp/pom.properties

**Comment by shelu16** [comment]
by 0x240x23elu

**Comment by princechaddha** [comment]
Hello @shelu16, Thank you for creating this issue. I appreciate your suggestion, but the existing endpoint is sufficient for the validation of this CVE, adding more paths would unnecessarily generate more traffic. If you disagree, please let us know.
